commit 8e4899affdd979174afed2a5a49865492e966590
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Tue Jun 19 17:20:11 2018 +0100

    Revert "configfs: hack: make it build"
    
    This reverts commit b2f722f44b4c73208c6bcdd8d32dc9b82bfacb77.

diff --git a/drivers/of/configfs.c b/drivers/of/configfs.c
index 9f608b2569da..178f0629b0f0 100644
--- a/drivers/of/configfs.c
+++ b/drivers/of/configfs.c
@@ -64,7 +64,7 @@ static int create_overlay(struct cfs_overlay_item *overlay, void *blob)
 	}
 	pr_debug("%s: resolved OK\n", __func__);
 
-	err = -1; // TODO: fix: of_overlay_apply(overlay->overlay, &overlay->ov_id);
+	err = of_overlay_apply(overlay->overlay, &overlay->ov_id);
 	if (err < 0) {
 		pr_err("%s: Failed to create overlay (err=%d)\n",
 				__func__, err);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1164_8e4899affdd979174afed2a5a49865492e966590.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1164_8e4899affdd979174afed2a5a49865492e966590.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
